### PR TITLE
MTV: added support for grouped season torrents

### PR DESCRIPTION
**Pull request by pkoutsias**
MTV groups season torrents and names them as 'Season X'. Added support in season and RSS search

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/providers/morethantv.py [hunk 1]
```diff
@@ -191,6 +256,18 @@ def process_column_header(td):
 
         return results
 
+    #Get search string in compatible 'Season x' mode
+    def _get_season_search_strings(self, episode):
```
Comment (miigotu) [inline sickbeard/providers/morethantv.py:260]: Overriding this is not allowed.

Comment (pkoutsias) [inline sickbeard/providers/morethantv.py:260]: Any suggestions?

**Comment by duramato** [comment]
https://www.morethan.tv/torrents.php?tags_type=1&order_by=time&order_way=desc&group_results=0&action=basic&searchsubmit=1

`group_results=0`

**Comment by kezek** [comment]
We should really find a viable solution to this , I would try to patch stuff up but my python skills are the worst. Any ideas ?

**Pull request closed without merging**